frontend/packages/api_client/src/models/DisplayStaticContentPages.ts

```diff
@@ -0,0 +1,97 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * OpenCraft Instance Manager
+ * API for OpenCraft Instance Manager
+ *
+ * The version of the OpenAPI document: api
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { exists, mapValues } from '../runtime';
+/**
+ * 
+ * @export
+ * @interface DisplayStaticContentPages
+ */
+export interface DisplayStaticContentPages {
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    about: boolean;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    contact: boolean;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    donate: boolean;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    tos: boolean;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    honor: boolean;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    privacyPolicy: boolean;
+}
+
+export function DisplayStaticContentPagesFromJSON(json: any): DisplayStaticContentPages {
+    return DisplayStaticContentPagesFromJSONTyped(json, false);
+}
+
+export function DisplayStaticContentPagesFromJSONTyped(json: any, ignoreDiscriminator: boolean): DisplayStaticContentPages {
+    if ((json === undefined) || (json === null)) {
+        return json;
+    }
+    return {
+        
+        'about': json['about'],
+        'contact': json['contact'],
+        'donate': json['donate'],
+        'tos': json['tos'],
+        'honor': json['honor'],
+        'privacyPolicy': json['privacy_policy'],
+    };
+}
+
+export function DisplayStaticContentPagesToJSON(value?: DisplayStaticContentPages | null): any {
+    if (value === undefined) {
+        return undefined;
+    }
+    if (value === null) {
+        return null;
+    }
+    return {
+        
```

Comment:
Same here, but I guess it would be worth asking, is there a reason you left an empty line before the first JSON item? 
If so, please ignore these suggestions

```suggestion
```